### DISCUSSION TITLE
saml2aws expects password for Azure AD Federated User

### DISCUSSION THREAD

**Discussion by TrueGrit16** [Versent/saml2aws#1421] [Q&A]
![img](https://github.com/Versent/saml2aws/assets/160136681/fa548cf3-8611-43c9-a74c-1588af0587d5)
saml2aws expects password for Azure AD Federated login. Does it work for scenario where the Azure AD authentication is using 2-digit number verification on Microsoft Authenticator?

**Comment by mapkon** [comment]
Have you tried it out?

**Comment by TrueGrit16** [comment]
Yes, and unable to use when password is prompted

**Comment by ioanvapi** [comment]
Switch from MS authenticator to Google one.
